specification/metrics/semantic_conventions/process-metrics.md

```diff
@@ -31,17 +32,18 @@ metrics](runtime-environment-metrics.md).
-| `process.memory.usage`          | UpDownCounter                                      | By        | The amount of physical memory in use.                                                                                               |                                                                                                                                                                                                 |
-| `process.memory.virtual`        | UpDownCounter                                      | By        | The amount of committed virtual memory.                                                                                             |                                                                                                                                                                                                 |
-| `process.disk.io`               | Counter                                            | By        | Disk bytes transferred.                                                                                                             | `direction` SHOULD be one of: `read`, `write`                                                                                                                                                   |
-| `process.network.io`            | Counter                                            | By        | Network bytes transferred.                                                                                                          | `direction` SHOULD be one of: `receive`, `transmit`                                                                                                                                             |
+| Name                            | Instrument Type ([\*](README.md#instrument-types)) | Unit     | Description                                                                                                                         | Labels                                                                                                                                                                                          |
+| ------------------------------- | -------------------------------------------------- | -------- | ----------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
My editor did this formatting I think, should I try to revert it? Not sure if this is expected.